configuration-sample/ods-core.env.sample

```diff
@@ -161,3 +161,54 @@ STORAGE_CLASS_DATA=""
 
 # Storage class for backup, for AWS this should be "gp2-encrypted"
 STORAGE_CLASS_BACKUP=""
+
+
+####################
+# Provisioning App #
+####################
+
+PROV_APP_ENVIRONMENT=cd
```

Comment:
Seeing the later use of this, it is the same as the automatic parameter `TAILOR_NAMESPACE`. Can we remove the sample param here, and make use of `TAILOR_NAMESPACE` in the templates?